index.js

```diff
@@ -90,6 +90,7 @@ raiseModal.forEach((trigger) => {
 
 closeModal.addEventListener('click', () => {
   modal.style.display = 'none';
+  window.location.reload(false);
```

Comment:
- [ ] All the project buttons are not opening popup please fix it :blush:
- [ ] On clicking the close button it should only close the popup. :+1: We don't need to refresh the page. Kindly remove the reload :+1:
- [ ] Project 2 is not showing the image please fix this issue :+1: